core/vm/gas_table.go

```diff
@@ -391,7 +393,7 @@ func gasCall(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize
 	} else if !evm.StateDB.Exist(address) {
 		gas += params.CallNewAccountGas
 	}
-	if transfersValue && !evm.chainRules.IsEIP4762 {
+	if transfersValue && (!evm.chainRules.IsEIP4762 || replayMode) {
```

Comment:
Other case that we should disable the effects.

Reply:
but isn't just deactivating eip 4762 enough?

Reply:
@gballet, https://github.com/gballet/go-ethereum/pull/436#discussion_r1593777953 :smile: